list_of_subprojects.md

```diff
@@ -0,0 +1,71 @@
+  - We propose that the Jupyer website and its repository will be governed by a working group that reports to the Board of Directors.
+  - We propose that the nbviewer service (only the actual live service) be governed by a working group that reports to the Board of Directors. The reusable part of nbviewer contained (https://github.com/jupyter/nbviewer) will be governed by the Jupyter Enterprise Subproject.
+- Jupyter Kernels
+  - To consolidate the project’s work on first-party kernels, we propose to establish a new official Subproject called _Jupyter Kernels_ and create a Github organization named jupyter-kernels for the work of the Subproject. All Xeus repositories (https://github.com/jupyter-xeus) will be moved into this organization. This Subproject will also govern the IPython GitHub organization, which will be left in place (https://github.com/ipython).
+  - A new decision making body for this organization will be established, and they will elect an SSC delegate.
+- Jupyter Standards
+  - Ultimately, because Jupyter standards are cross-project in nature, they are owned by the SSC. The mechanics of day-to-day management of the JEP repo will be decided by the SSC.
+  - There are a number of repositories that encode project-wide standards and protocols. To consolidate work on these repositories, we propose to establish a new official Subproject called Jupyter Standards and create a Github organization named jupyter-standards for the work of the Subproject. The following repositories will be moved into this organization:
+    - Jupyter Client (https://github.com/jupyter/jupyter_client)
+    - Jupyter Notebook Format (https://github.com/jupyter/nbformat)
+    - Documentation for other specifications maintained by other Subprojects, such as the Jupyter Widgets message specification, or the Jupyter Server REST APIs.
+    - JEPs repo (https://github.com/jupyter/enhancement-proposals).
+- Jupyter Enterprise
+  - A number of Jupyter repositories pertain to enterprise deployments. To consolidate work on these repositories, we propose to establish a new official Subproject called _Jupyter Enterprise_ and create a Github organization named jupyter-enterprise for the work of the Subproject. The following repositories will be moved into this organization:
+  - Enterprise Gateway (https://github.com/jupyter/enterprise_gateway)
+  - Kernel Gateway (https://github.com/jupyter/kernel_gateway)
+  - Docker stacks (https://github.com/jupyter/docker-stacks)
+  - Jupyter Notebook Viewer (https://github.com/jupyter/nbviewer)
+- Jupyter Foundations
```

Comment:
This might confuse people, when I first saw this I thought that Jupyter was proposing to create legal foundations for each of these

Reply:
Sorry for the slight confusion but we can't think of a better term now, and this one does convey clearly the intent, so hopefully with usage the double-take will fade :)